storage/azure/src/server/AzureServerStorage.ts

```diff
@@ -143,6 +157,10 @@ export class AzureServerStorage extends ServerStorage {
     );
   }
 
+  public async list(directory: BaseDirectory): Promise<ObjectReference[]> {
```

Comment:
Maybe would be good to mark implementations as deprecated too? In case someone is using these classes directly